jobs/config.json

```diff
@@ -8478,8 +8478,8 @@
       "--extract=ci/latest",
       "--gcp-zone=us-central1-f",
       "--provider=gce",
-      "--test_args=--ginkgo.focus=Variable.Expansion --minStartupPods=8",
-      "--timeout=40m"
+      "--test_args=--ginkgo.skip=\\[Slow\\]|\\[Serial\\]|\\[Disruptive\\]|\\[Flaky\\]|\\[Feature:.+\\] --minStartupPods=8",
```

Comment:
This could make it run quite long. Are you fine having the delay?

Reply:
+20~30min is fine - especially now as gce-canary is borked by gcloud, I want some gce signal for the kubetest changes.